### PR TITLE
Implemented the CDS Term of the SMD

### PR DESCRIPTION
**Pull request by MtoLStoN**
This will implement the cds term of the SMD with default parameters and the possibility to define own SMD Parameters.

### CODE REVIEW

**Comment by MtoLStoN** [comment]
Should work now with default SMD Parameters for DMSO, Methanol, Acetonitrile and H2O. Also adds the possibility to use any Solvent by adding a property file *.prop into the working directory. Adding own Parameters for the SMD for Fitting Purposes etc. will follow.

**Review by awvwgk — CHANGES_REQUESTED** [review]
Thanks, this looks overall well done, but I'm missing at least some testing. Can you add some regression testing for this new feature in the `test` subdirectory?

Gradients would be nice as well, but we can look into this separately.

File: app/main.f90 [hunk 1]
```diff
@@ -91,6 +99,13 @@ program main_driver
 
    call ascii_surface_area(output_unit, mol, surface)
 
+   if (allocated(config%solvent)) then
+      Call init_smd(param,config%solvent)
+      Call calc_surft(mol%xyz,mol%id,mol%sym,param,surft)
+      Call calc_cds(surft,surface,mol%sym,mol%id,cds,cds_sm)
+      Call ascii_cds(output_unit,mol,cds,cds_sm)
```
Comment (awvwgk) [inline app/main.f90]:
```suggestion
      call init_smd(param,config%solvent)
      call calc_surft(mol%xyz,mol%id,mol%sym,param,surft)
      call calc_cds(surft,surface,mol%sym,mol%id,cds,cds_sm)
      call ascii_cds(output_unit,mol,cds,cds_sm)
```

File: test/test_cds.f90 [hunk 2]
```diff
@@ -0,0 +1,189 @@
+
```
Comment (awvwgk) [inline test/test_cds.f90]:
```suggestion
```

**Review by awvwgk — APPROVED** [review]
Thanks, look good now.

**Pull request merged**